exp-player/addon/randomizers/pref-phys.js

```diff
@@ -10,105 +12,1336 @@ function shuffleArray(array) {
+        ['tissues', 'shoe', 'hammer', 'book', 'duck', 'brush'],
+        ['hammer', 'duck', 'book', 'shoe', 'brush', 'tissues'],
+        ['brush', 'duck', 'hammer', 'shoe', 'book', 'tissues'],
+        ['duck', 'shoe', 'tissues', 'book', 'brush', 'hammer'],
+        ['tissues', 'brush', 'hammer', 'duck', 'shoe', 'book'],
+        ['hammer', 'book', 'shoe', 'tissues', 'brush', 'duck'],
+        ['duck', 'tissues', 'shoe', 'brush', 'book', 'hammer'],
+        ['book', 'brush', 'shoe', 'hammer', 'tissues', 'duck'],
+        ['shoe', 'brush', 'tissues', 'duck', 'book', 'hammer'],
+        ['shoe', 'hammer', 'brush', 'book', 'duck', 'tissues'],
+        ['shoe', 'brush', 'book', 'hammer', 'duck', 'tissues'],
+        ['hammer', 'duck', 'shoe', 'tissues', 'brush', 'book'],
+        ['duck', 'brush', 'book', 'shoe', 'tissues', 'hammer'],
+        ['brush', 'tissues', 'book', 'shoe', 'duck', 'hammer'],
+        ['brush', 'hammer', 'tissues', 'book', 'shoe', 'duck'],
+        ['shoe', 'hammer', 'tissues', 'brush', 'duck', 'book'],
+        ['duck', 'brush', 'tissues', 'shoe', 'hammer', 'book'],
+        ['brush', 'book', 'tissues', 'hammer', 'shoe', 'duck'],
+        ['duck', 'hammer', 'book', 'brush', 'shoe', 'tissues'],
+        ['tissues', 'shoe', 'duck', 'brush', 'hammer', 'book'],
+        ['duck', 'brush', 'hammer', 'tissues', 'book', 'shoe'],
+        ['shoe', 'book', 'hammer', 'brush', 'tissues', 'duck'],
+        ['book', 'duck', 'tissues', 'shoe', 'hammer', 'brush'],
+        ['tissues', 'duck', 'book', 'shoe', 'hammer', 'brush'],
+        ['hammer', 'book', 'duck', 'brush', 'shoe', 'tissues'],
+        ['tissues', 'brush', 'shoe', 'duck', 'book', 'hammer'],
+        ['duck', 'brush', 'tissues', 'hammer', 'book', 'shoe'],
+        ['brush', 'hammer', 'shoe', 'book', 'tissues', 'duck'],
+        ['hammer', 'brush', 'shoe', 'tissues', 'book', 'duck'],
+        ['book', 'brush', 'shoe', 'tissues', 'hammer', 'duck'],
+        ['hammer', 'book', 'duck', 'tissues', 'brush', 'shoe'],
+        ['brush', 'book', 'duck', 'hammer', 'shoe', 'tissues'],
+        ['hammer', 'tissues', 'brush', 'book', 'duck', 'shoe'],
+        ['duck', 'shoe', 'brush', 'book', 'tissues', 'hammer'],
+        ['duck', 'hammer', 'tissues', 'brush', 'book', 'shoe'],
+        ['shoe', 'duck', 'tissues', 'hammer', 'book', 'brush'],
+        ['hammer', 'book', 'brush', 'duck', 'tissues', 'shoe'],
+        ['hammer', 'shoe', 'tissues', 'book', 'duck', 'brush'],
+        ['book', 'brush', 'tissues', 'duck', 'shoe', 'hammer'],
+        ['book', 'brush', 'duck', 'shoe', 'hammer', 'tissues'],
+        ['shoe', 'duck', 'hammer', 'book', 'tissues', 'brush'],
+        ['hammer', 'tissues', 'duck', 'brush', 'shoe', 'book'],
+        ['duck', 'book', 'brush', 'shoe', 'tissues', 'hammer'],
+        ['duck', 'shoe', 'book', 'hammer', 'tissues', 'brush'],
+        ['shoe', 'brush', 'hammer', 'duck', 'tissues', 'book'],
+        ['brush', 'tissues
```

Comment:
We try to minimize console.log statements and often remove local debugging code before merging; do we want to keep this statement in? 

Also, if this is for more than just local debugging: Is this meant to be a likely and tolerable error that we can ignore, or is this an event that we explicitly want to see logged to sentry?